MAUI/Accordion/BindableLayout.md

```diff
@@ -0,0 +1,295 @@
+---
+layout: post
+title: BindableLayout | Accordion for .NET MAUI | Syncfusion
+description: This section described about how to bind AccordionItem to .NET MAUI Accordion (SfAccordion) using BindableLayout.
```

Comment:
Description: This section describes how to bind AccordionItem to .NET MAUI Accordion (SfAccordion) using BindableLayout.

Reply:
Here, description should be in small.